sources/platform/actors/publishing/index.mdx

```diff
@@ -5,39 +5,88 @@ sidebar_position: 7.5
 slug: /actors/publishing
 ---
 
-**Learn about publishing, and monetizing your Actors on the Apify platform.**
+**Apify provides a platform for developing, publishing, and monetizing web automation solutions called Actors. This guide covers the key stages involved in publishing and monetizing your Actors on the Apify platform.**
 
 ---
 
 > Sharing is caring but you can also make money from your Actors. Our [blog post](https://blog.apify.com/make-regular-passive-income-developing-web-automation-actors-b0392278d085/) contains all the information you need.
 
-To build a public Actor, you'll go through the following four main stages:
+
+## Publish process
+
+To build & publish a public Actor on Apify platform, you'll go through the following main stages:
 
 1. [Development](../development/index.md).
 2. [Publication](./publishing/publish) and set up of [monetization](./publishing/monetize).
 3. [Testing](../development/automated_tests.md).
 4. [Promotion](../../academy/get-most-of-actors/seo-and-promotion).
 
-While you don't necessarily have to maintain your private Actors, public Actors require a higher degree of responsibility.
+## Benefits of publishing and monetization
+
+Publishing your Actor on the Apify Store transforms your code into a sustainable revenue stream while eliminating traditional SaaS development overhead. When you publish your Actor, you automatically receive:
+
+- A dedicated landing page showcasing your Actor's capabilities
+- Built-in documentation hosting through README integration
+- Direct exposure to organic user traffic
+- Flexible sharing options with public or private source code settings
+- Complete API documentation and integration examples
+
+<details>
+    <summary>Publishing on Apify significantly streamlines the traditional SaaS development process</summary>
+    | Traditional SaaS Requirements | Handled by Apify Platform |
+    |------------------------------|---------------------------|
+    | Product development | ✓ Actor development only |
+    | Extensive documentation | ✓ README file |
+    | Domain purchase & setup | ✓ Automatic landing page |
+    | Cloud infrastructure | ✓ Built-in scaling |
+    | Payment processing | ✓ Multiple payment options |
+    | Marketing campaigns | ✓ Store marketplace presence |
+    | Sales operations | ✓ Automated transactions |
+</details>
+
+### Monetization options
+
+The platform supports various pricing models to match your Actor's value proposition:
 
-As the name implies, Public Actors are available to the public on [Apify Store](https://apify.com/store), which means that an unmaintained public Actor could negatively affect all the other users that depend on it for their own activities on the platform.
+- Fixed rental fee for continuous access
+- Pay-per-result for usage-based pricing
+- Event-based pricing for specific operations
+- Custom pricing strategies for enterprise clients
```

Comment:
I'd skip the "custom pricing", it's too confusing and not really available to the devs

Reply:
BTW can we link some page with details about the pricing models? and maybe provide some examples?

Reply:
Also we call it "pay per event", not "event-based pricing"

Reply:
The closest we got that could serve as more context for pricing models would be [this](https://docs.apify.com/platform/actors/running/actors-in-store#pricing-models), I'll add it to the doc